<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>831</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>设置</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>216</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>扳手名称:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineName"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>216</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>开门扭矩:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Linekaimen"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>216</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>松开扭矩:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineSongkai"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>216</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>目标角度:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineMubiaoJiao"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>216</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>角度下限:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineJiaoduX"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>216</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>角度上限:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineJiaoduS"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>216</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>目标扭矩:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineMubiaoNiu"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>216</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>扭矩下限:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineNiujuX"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>216</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>扭矩上限:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineNiujuS"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>192</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>方  向:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="CW">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>正向</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="CCW">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>反向</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>192</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>模  式:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mode_a">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>角度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mode_n">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>扭矩</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="DownLoad">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>275</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>下载</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>821</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>接收</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
